### DISCUSSION TITLE
`pcre2-sys` crate build failed in linux

### DISCUSSION THREAD

**Discussion by Peefy** [messense/maturin-action#261] [Q&A] [answered]
Reference: https://github.com/kcl-lang/lib/actions/runs/8658577506/job/23742724269?pr=53

![image](https://github.com/PyO3/maturin-action/assets/22744597/de2f421c-0533-433e-bb48-0b81bb1caac6)

**Comment by messense** [comment, marked as answer]
Seems that pcre2 requires a newer GCC version, you should target higher manylinux version to get a newer GCC.
For example when cross compiling `manylinux_2_28` has GCC 7.5.0 while `manylinux2014` only has GCC 4.8.5.

See https://github.com/rust-cross/manylinux-cross?tab=readme-ov-file#manylinux-cross
